app/controllers/api/v1/monsters_controller.rb

```diff
@@ -19,9 +21,26 @@ def create
     end
   end
 
-  private
+  def destroy
+    Monster.find(params[:id]).destroy
+    render json: {}, status: :no_content
+  end
+
+  protected 
```

Comment:
Let's change this back to `private` indicating the methods below are for this controller only